docs/source/user_guide/pipeline-components.md

```diff
@@ -28,15 +28,21 @@ The same pipeline could be implemented using a single component that performs al
 
 #### Generic components
 
-Elyra includes three _generic components_ that allow for the processing of Jupyter notebooks, Python scripts, and R scripts. These components are called generic because they can be included in pipelines for any supported runtime type: local/JupyterLab, Kubeflow Pipelines, and Apache Airflow. Components are exposed in the pipeline editor via the palette.
+Viewed
```

Comment:
@shalberd is this a conflict?

Reply:
yes, it looks like a rebase tool merge conflict, resolving it now. Thank you, resolving it now.